koa-pug/index.d.ts

```diff
@@ -35,3 +33,16 @@ export namespace Pug {
     }
     export interface PugLocals { [str: string]: any; }
 }
+
+
+
+/** plugin */
+declare module 'koa' {
+    interface Context {
+        render(viewPath: string, locals?: Pug.PugLocals, overrideOpts?: Pug.PugCtxOptions, noCache?: boolean): void;
+        // tslint:disable-next-line:unified-signatures
+        render(pugStr: string, locals?: Pug.PugLocals, overrideOpts?: Pug.PugCtxOptions): void;
```

Comment:
There's no way to use this overload. If you pass in `string, PugLocals, PugCtxOptions` it will be considered to be the first overload (with `noCache` left out).